RibonDAO/interface
# Tech 563: get chains and remove networks file

# Description

- get chains from backend and remove networks file

# Necessary changes

- update environment variables

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Make a crypto contribution
